channels/jo.m3u

```diff
@@ -7,9 +7,11 @@ https://ammantv.c.s73cdn.net/23153d43-375a-472a-bc5f-9827582b5d22/elemental/live
 http://two.alfajertv.com:8081/AlfajertvHDTwo_web/web/playlist.m3u8
 #EXTINF:-1 tvg-id="JawharaFM.jo" tvg-name="Jawhara FM" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://www.jawharafm.net/ar/static/fr/image/png/logo-jfm.png" group-title="Music",Jawhara FM (720p)
 http://streaming.toutech.net:1935/live/mp4:jawharafm.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="Jordan TV" tvg-name="Jordan TV" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://i.imgur.com/siAiiws.jpg" group-title="General",Jordan TV
+#EXTINF:-1 tvg-id="Jordan TV" tvg-name="Jordan TV" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://static.jrtv.gov.jo/resize-width/380/content/x/Mp/1/Poster.png" group-title="General",Jordan TV
 https://jrtv-live.ercdn.net/jordanhd/jordanhd.m3u8
-#EXTINF:-1 tvg-id="MelodyFM.jo" tvg-name="Melody FM" tvg-country="JO" tvg-language="Arabic" tvg-logo="http://melody-fm.com/img/Logo.png" group-title="Music",Melody FM (720p)
+#EXTINF:-1 tvg-id="Jordan TV" tvg-name="Jordan TV" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://static.jrtv.gov.jo/resize-width/380/content/y/B5/2/Poster.png" group-title="Sports",Jordan Sport [Offline]
+https://jrtv-live.ercdn.net/jordansporthd/jordansporthd.m3u8
```

Comment:
This might be Geo-blocked. I've marked it as Offline

Reply:
401 Auth required